Nitrox.Launcher/Models/Design/ServerEntry.cs

```diff
@@ -107,49 +110,66 @@ protected override void OnPropertyChanged(PropertyChangedEventArgs e)
 
     public static ServerEntry CreateNew(string saveDir, NitroxGameMode saveGameMode)
     {
+        ArgumentException.ThrowIfNullOrWhiteSpace(saveDir, nameof(saveDir));
+
         Directory.CreateDirectory(saveDir);
+
         SubnauticaServerConfig config = SubnauticaServerConfig.Load(saveDir);
-        string fileEnding = "json";
-        if (config.SerializerMode == ServerSerializerMode.PROTOBUF)
+        string fileEnding = config.SerializerMode switch
         {
-            fileEnding = "nitrox";
-        }
+            ServerSerializerMode.JSON => "json",
+            ServerSerializerMode.PROTOBUF => "nitrox",
+            _ => throw new NotImplementedException()
+        };
 
         File.WriteAllText(Path.Combine(saveDir, $"Version.{fileEnding}"), null);
```

Comment:
Should prefix `fileEnding` with `.`. Then remove the . in version file write.

Reply:
Whoops, forgot to bring the changes. I did it few lines later